registrationPage.scss

```diff
@@ -0,0 +1,61 @@
+@mixin centering-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.registrationPageContainer {
+  @include centering-container;
+  max-width: 450px;
+  width: 100%;
+  margin: 0 auto;
+  background: #D1D8E2;
+  position: relative;
+  top: 100px;
+  border-radius: 25px;
+  .createUserIntro {
+
+    text-align: left;
+    font-size: 1.5rem;
+    color: #000000;
+  }
+
+  .registrationForm {
+    @include centering-container;
+    max-width: 300px;
+    width: 300px;
+    gap: 10px;
+    margin: 0 auto;
+  }
+  .submitForm {
```

Comment:
change this class to be called `submitFormButton` so it doesn't look like it's a form

Reply:
Fixed.